CONTRIBUTING.md

```diff
@@ -149,17 +149,27 @@ Releases are cut with `git` tags and a successful Travis build results in new bi
 
 Credentials for the `openfaas` and/or `functions` Docker Hub and quay.io accounts are coordinated by the project lead using.
 
+## Governance
+
+OpenFaaS is an independent open-source project which was created by the Project Lead Alex Ellis in 2016. OpenFaaS is now being built by Alex, a number of volunteer teams, and a wider community of open-source developers.
```

Comment:
Perhaps the use of "Project Lead" should be highlight here or defined somewhere else so that is is easier to find as a definition because it is used in `#### What happens if you ignore the process?`

Reply:
I am going to comment out that section for the time-being. I want to revisit the tone.

Reply:
I've updated that section, if you could take another look.